app/bounty_requests/views.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+"""Define bounty request views.
+
+Copyright (C) 2018 Gitcoin Core
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+import json
+
+from django.http import JsonResponse
+from django.template.response import TemplateResponse
+from django.utils.html import escape, strip_tags
+from django.utils.translation import gettext_lazy as _
+
+from marketing.mails import new_bounty_request
+from ratelimit.decorators import ratelimit
+
+from .models import BountyRequest
+
+
+@ratelimit(key='ip', rate='5/m', method=ratelimit.UNSAFE, block=True)
+def bounty_request(request):
+    user = request.user if request.user.is_authenticated else None
+    profile = request.user.profile if user and hasattr(request.user, 'profile') else None
+
+    if request.body:
+        if not user or not profile or not profile.handle:
+            return JsonResponse(
+                {'error': _('You must be Authenticated via Github to use this feature!')},
+                status=401)
+
+        try:
+            params = json.loads(request.body)
+        except Exception:
+            return JsonResponse({'error': 'Invalid JSON.'}, status=400)
+
+        # requested_by = params.get('requested_by', '')
+        requested_by = profile.handle
```

Comment:
If you update `requested_by` to a FK, you can simply ref the profile or profile id here.